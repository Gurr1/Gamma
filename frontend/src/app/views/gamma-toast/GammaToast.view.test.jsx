import React from "react";
import { shallow } from "enzyme";
import GammaToast from "./GammaToast.view";

describe("<GammaToast/>", () => {
  test("Shallow render of <GammaToast/>", () => {
    const wrapper = shallow(<GammaToast />);
    expect(wrapper).toMatchSnapshot();
  });
});
