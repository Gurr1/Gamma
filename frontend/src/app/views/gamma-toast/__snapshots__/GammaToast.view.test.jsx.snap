// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GammaToast/> Shallow render of <GammaToast/> 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GammaToast />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Styled(WithStyles(Snackbar))
        ContentProps={
                Object {
                        "aria-describedby": "message-id",
                      }
        }
        action={
                <Styled(Component)
                        color="secondary"
                        hide={false}
                        onClick={[Function]}
                        size="small"
                />
        }
        anchorOrigin={
                Object {
                        "horizontal": "left",
                        "vertical": "bottom",
                      }
        }
        autoHideDuration={0}
        message={
                <span
                        id="message-id"
                />
        }
        onClose={[Function]}
        onExited={[Function]}
        open={false}
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "ContentProps": Object {
          "aria-describedby": "message-id",
        },
        "action": <Styled(Component)
          color="secondary"
          hide={false}
          onClick={[Function]}
          size="small"
/>,
        "anchorOrigin": Object {
          "horizontal": "left",
          "vertical": "bottom",
        },
        "autoHideDuration": 0,
        "message": <span
          id="message-id"
/>,
        "onClose": [Function],
        "onExited": [Function],
        "open": false,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Styled(WithStyles(Snackbar))
          ContentProps={
                    Object {
                              "aria-describedby": "message-id",
                            }
          }
          action={
                    <Styled(Component)
                              color="secondary"
                              hide={false}
                              onClick={[Function]}
                              size="small"
                    />
          }
          anchorOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "bottom",
                            }
          }
          autoHideDuration={0}
          message={
                    <span
                              id="message-id"
                    />
          }
          onClose={[Function]}
          onExited={[Function]}
          open={false}
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "ContentProps": Object {
            "aria-describedby": "message-id",
          },
          "action": <Styled(Component)
            color="secondary"
            hide={false}
            onClick={[Function]}
            size="small"
/>,
          "anchorOrigin": Object {
            "horizontal": "left",
            "vertical": "bottom",
          },
          "autoHideDuration": 0,
          "message": <span
            id="message-id"
/>,
          "onClose": [Function],
          "onExited": [Function],
          "open": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
